Fix RC discovery & use WebSocket to check for changed status

aPinat opened the pull request:
- Using List instead of Array for checking RC paths
- Support .NET 4.6+ (version released and bundled with the first release of Win10 in 2015, so old enough to not prompt any Win10 users to update .NET, while still being fully functional)
- check for system.yaml in LCU path instead of Logs folder (for new/never before run LCU installs)
- use WebSocket to monitor chat status changes and update the in-client status to the masked status, making it much more reliable

molenzwiebel commented on Deceive/MainController.cs:
```diff
@@ -215,7 +213,8 @@ private void UpdateStatus(string status)
             this.PossiblyRewriteAndResendPresence(this.lastPresence, status);
-            Utils.SendStatusToLCU(status);
+            _ws.Close();
+            Utils.MonitorChatStatusChange(status);
```
Shouldn't this be a `_ws = Utils.MonitorChatStatusChange(status)`?

molenzwiebel commented on Deceive/Utils.cs:
```diff
@@ -276,21 +274,44 @@ public static LCUAPIPortToken GetAPIPortAndToken(Process process)
+        {
+            foreach (var process in Process.GetProcessesByName("LeagueClientUx"))
+            {
+                var apiAuth = GetApiPortAndToken(process);
```
Should probably for completeness sake check if apiAuth != null

aPinat commented:
Implemented both, removed some redundant code along the way.
Had to figure out, how pushing the same changes to different branches works :-D
(Build: https://github.com/aPinat/Deceive/releases/tag/20191120.1)
